docs/reference/commandline/run.md

```diff
@@ -406,7 +406,6 @@ USER
 $ docker run --env-file env.list ubuntu env | grep VAR
```

Comment:
So, the intent of this example was to show the environment variables, but only the relevant ones for this example, and to exclude env-vars like `$PATH` or `$HOSTNAME`.

Perhaps instead we should change the `grep` command;

```bash
$ docker run --env-file env.list ubuntu env | grep -E 'VAR|USER'
```